python-backend/app/api/incidents/models/mine_incident_recommendation.py

```diff
@@ -5,8 +5,10 @@
 from sqlalchemy.dialects.postgresql import UUID
 from sqlalchemy.schema import FetchedValue
 from app.extensions import db
-from ....utils.models_mixins import AuditMixin, Base
 
+from app.api.utils.models_mixins import AuditMixin, Base
+
+INVALID_MINE_INCIDENT_RECOMMENDATION_GUID = 'Invalid mine_incident_recommendation_guid.'
```

Comment:
It doesn't seem useful to me to pull validation messages out as variables. What am I not seeing?

Reply:
I did this with Variances for the purpose of importing the constant(s) into the tests to assert the correct validation error.

There is a lot of missing test coverage, so I haven't added that in this PR.

There are potentially cons to having this split out into a constant, so I'm open to putting the string directly in the response if there are concerns. I think it's a net win, but that's only my perspective.

Reply:
Ok that makes sense. Asserting validation messages and the general overhead of string constants is a feel bad to me personally, but if the tests and API are using the same constants and we need to validate more granular than our response codes allow I do get it.

Reply:
I'm not sure that we _need_ to validate these strings. My reason for doing so was that we have had issues with useless errors and/or exception stack trace dumps in the responses historically, so I liked knowing that we had some coverage on that.

With the fix for global error handlers last month, this may no longer be an issue. At which point, we can remove the feel bad of validating error strings? 🙂